mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/course/ChapterContentMenu.java

```diff
@@ -0,0 +1,44 @@
+package org.motechproject.nms.mobileacademy.domain.course;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Contains the details about the menu file to be played at the end of the chapter
+ */
+@Entity
+public class ChapterContentMenu {
+
+    /**
+     * This is id for the End menu file of the chapter in the format”Chapter<ChapterId>_EndMenu”, where
+     * chapterId varies from 01 to 11.
+     */
+    @Field
+    private String identifier;
+
```

Comment:
+1 on Rob's rec -- and then use similar naming for the other Entity classes that have "identifier" members.